### ISSUE TITLE
webpack issue

### ISSUE THREAD

**Issue by stevermeister** [IamAdamJowett/angular-click-outside#38]
trying to build this module with npm and webpack 

    import '@iamadamjowett/angular-click-outside';

and getting error:

    Module not found: Error: Cannot resolve module '@iamadamjowett/angular-click-outside'

**Comment by vlki** [comment]
I had this issue with angular-click-outside 2.7.0, because it does not have the `main` property in package.json. That was added as part of #36 and is in version 2.7.1.

Version 2.7.1 is not published to npm as I write this comment, but is tagged here in github repo. Before the version gets published to npm, I am installing it directly from github repo by 
```
npm install iamadamjowett/angular-click-outside#v2.7.1 --save
```

**Comment by IamAdamJowett** [comment]
@stevermeister @vlki I have published the latest version v2.8.2 to npm which contains this fix also, thanks for pointing this out. 5c64f22
